app/dashboard/templatetags/bundle.py

```diff
@@ -113,8 +113,8 @@ def check_for_changes(elems, attr, kind, outputFile):
             if el.get(attr):
                 # check if we're loading an alternative source in production
                 file = el[attr]
-                if isProduction and el.get('prod'):
-                    file = el['prod']
+                if assetsHosted and el.get(settings.ENV):
+                    file = el[settings.ENV]
```

Comment:
This is a prop on the import to allow us to select a different asset in `prod` vs `dev` - check out how its being used here: https://github.com/gitcoinco/web/blob/e0fdf7704ba8deb8ee5fba9ca4161540b3d69521/app/retail/templates/shared/footer_scripts.html#L34